src/files-action.js

```diff
@@ -42,3 +44,21 @@ window.addEventListener('DOMContentLoaded', () => {
 	}
 	console.error('Unable to register vcf import action')
 })
+
+if (nextcloudVersionIs28) {
+	registerFileAction(new FileAction({
+		name,
+		displayName() {
+			return t('contacts', 'Import')
+		},
+		default: 'default',
+		mime,
+		permissions: OC.PERMISSION_READ,
```

Comment:
I'd prefer if this came from `@nextcloud/files` and not a global variable of server